src/main/java/io/github/vampirestudios/raa/api/enums/GeneratesIn.java

```diff
@@ -4,32 +4,32 @@
 import net.minecraft.block.Block;
 import net.minecraft.block.Blocks;
 
-public class GeneratesIn {
```

Comment:
change this back to a class, since we need it to be a class when we're gonna add support for dimensional ores